src/reducers/screen.ts

```diff
@@ -298,11 +300,15 @@ export function screen(state = initialState, action: AnyAction): ScreenState {
         }
         case types.bcAddFilter: {
             const { bcName, filter } = action.payload
+            const isDate = store.view.widgets
+                .find(item => item.name === action.payload.widgetName)
+                ?.fields.find((item: WidgetField) => item.type === FieldType.date && item.key === action.payload.filter.fieldName)
```

Comment:
Let's use `filter` from already destructured `action.payload` object. 
I mean `item.key === filter.fieldName`

Reply:
Fixed in https://github.com/tesler-platform/tesler-ui/pull/688/files